File: Scripts/Matlab/RBFp2Trj.m
function y = RBFp2Trj(RBF)
%% Trajektorijo q realiziram z DMP-ji
y = [];
S.x = 1;
while (S.x >=  exp(-RBF.a_x))
   [S]=RBFp_Integrate(RBF,S);
   y = [y;S.y];
end 

File: Scripts/Python/RBFp2Trj.py
import numpy as np
from RBFp_Integrate import RBFp_Integrate
def RBFp2Trj(RBF):
    y = []
    S = {"x": 1}

    while S["x"] >= np.exp(-RBF["a_x"]):
        S = RBFp_Integrate(RBF, S)  # Assuming RBFp_Integrate is another function you might provide later
        y.append(S["y"])

    y = np.array(y)
    return y